lego/apps/events/models.py

```diff
@@ -253,6 +255,9 @@ def register(self, registration):
         if self.registration_close_time < current_time:
             raise EventHasClosed()
 
+        if self.share_info_flag and user.phone_number is None:
```

Comment:
Should make some logic to handle this and get the error msg to the client, and show it. This will currently fail bad, and the user will keep seeing the loading indicator. (same applies to the other errors as well tho.)

Maybe outside the scope of this tho.

Reply:
Should be fixed to all cases. Thinking this is not that important to fix here since the admitt button is hidden in the frontend anyways.